xASL_qc_ReportLeftRightFlips.m

```diff
@@ -0,0 +1,21 @@
+% Check for left-right flips for all NIfTIs inside a folder
```

Comment:
This file shouldn't be in the main directory of ExploreASL

Reply:
Had another look. It seems as if there is also another function with the same name under `Functions`. Maybe move this to `CustomScripts` because of the fixed paths etc.?

Reply:
Yes you're right, fixed this now in last commits